src/System.Windows.Forms/tests/AccessibilityTests/ToolStripContainer.Designer.cs

```diff
@@ -267,7 +267,7 @@ private void InitializeComponent()
             // 
             // toolStripComboBox1
             // 
-            this.toolStripComboBox1.AccessibleName = "ComboBox";
+            this.toolStripComboBox1.AccessibleName = "ComBo_Box";
```

Comment:
This change looks inconsistent with changes in https://github.com/dotnet/winforms/pull/3707/files#diff-4f5a4f45713cb61b5de252da8ecdb6ee
```suggestion
            this.toolStripComboBox1.AccessibleName = "TSCB";
```